### PR TITLE
Persons for ef116

### PR DESCRIPTION
**Pull request by DariaElagina**
New and updated entries

### CODE REVIEW

File: PRS3001-4000/PRS3521Denqenas.xml [hunk 1]
```diff
@@ -44,26 +44,59 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                  Dǝnqǝnäš Mǝrḉa
-               </persName>
-               <occupation> ǝtege </occupation>
+               <persName xml:id="n1" xml:lang="am">ድንቅነሽ፡ ምርጫ፡</persName>
+               <persName corresp="#n1" xml:lang="am" type="normalized">Dǝnqǝnaš Mǝrč̣ā</persName>
+               <persName><roleName type="title">ʾǝtege</roleName></persName>
+               <occupation type="political"></occupation>
+               <birth when="1833"></birth>
+               <floruit>
+                 Dǝnqǝnaš Mǝrč̣ā was first married to <persName ref="PRS4258GabraKi"></persName> from whom he had
```
Comment (thea-m) [inline PRS3001-4000/PRS3521Denqenas.xml]: she had

File: PRS3001-4000/PRS3521Denqenas.xml [hunk 2]
```diff
@@ -44,26 +44,59 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                 a son, <persName ref="PRS8799seyyum"></persName>, and three daughters,
+                 <persName ref="PRS1996Aradacc"></persName>, <persName ref="PRS9132Takkolac"></persName>, and
+                 <persName ref="PRS2759BezuGab"></persName>. After her divorce, she married
+                 <persName ref="PRS9143TaklaGi">Gobaze Gabra Madḫən</persName> in <date>1867</date>. The later proclaimed himself
```
Comment (thea-m) [inline PRS3001-4000/PRS3521Denqenas.xml]: He later or the latter?

File: PRS3001-4000/PRS3521Denqenas.xml [hunk 3]
```diff
@@ -44,26 +44,59 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                  ʾaṣe Takla Giyorgis II in <date>1867</date>, thus Dǝnqǝnaš Mǝrč̣ā received a title of ʾǝtege. Afterthe defeat of her
```
Comment (thea-m) [inline PRS3001-4000/PRS3521Denqenas.xml]: After the

File: PRS6001-7000/PRS6668Mangasa.xml [hunk 4]
```diff
@@ -51,19 +51,37 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
       <body>
          <listPerson>
             <person sex="1">
-               <persName>
-                  
-                  Mängäša Yoḥannǝs
-               </persName>
-               <occupation> ras </occupation>
+              <persName xml:id="n1" xml:lang="am">መንገሻ፡ ዮሐንስ፡</persName>
+               <persName corresp="#n1" xml:lang="am" type="normalized">Mangašā Yoḥānnǝs</persName>
```
Comment (thea-m) [inline PRS6001-7000/PRS6668Mangasa.xml]: ḥa

File: PRS6001-7000/PRS6668Mangasa.xml [hunk 5]
```diff
@@ -51,19 +51,37 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
          <change who="PL" when="2016-03-21"> Created file from
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: place</change>
+         <change when="2020-01-07" who="DE">Added information and bibliographz.</change>
```
Comment (thea-m) [inline PRS6001-7000/PRS6668Mangasa.xml]: bibliography

File: PRS7001-8000/PRS7019MercaWa.xml [hunk 6]
```diff
@@ -44,26 +44,53 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+               <persName xml:id="n1" xml:lang="am">ምርጫ፡ ወርቄ፡
                </persName>
-               <occupation> liq.m. </occupation>
+               <persName corresp="#n1" type="normalized">Mǝrč̣ā Warqe</persName>
+              <persName xml:lang="en" corresp="#n1">M. Warkee</persName>
+               <persName><roleName type="title">liqa makʷās</roleName></persName>
+               <persName><roleName type="title">lǝǧ</roleName></persName>
+               <occupation type="political"></occupation>
+               <birth when="1825"></birth>
+               <floruit>He was one of the first foreign-educated Ethiopians who acted as a diplomat, secretarz and interpreter.
```
Comment (thea-m) [inline PRS7001-8000/PRS7019MercaWa.xml]: secretary

File: PRS7001-8000/PRS7019MercaWa.xml [hunk 7]
```diff
@@ -44,26 +44,53 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+Mǝrč̣ā Warqe received his education at the British missionary school of John Wilson in <placeName ref="LOC1956Bombay">Bombay</placeName>.
+In <date>1849</date> he left India and founded a school in <placeName ref="LOC1264Adwa"></placeName> to gether with his
```
Comment (thea-m) [inline PRS7001-8000/PRS7019MercaWa.xml]: together

File: PRS7001-8000/PRS7019MercaWa.xml [hunk 8]
```diff
@@ -44,26 +44,53 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+brother <persName ref="PRS4398GabruWa"></persName>. Later, Mǝrč̣ā Warqe became a follower of <persName ref="PRS9430Tewodros"></persName>.
+He played a crucial political role during the British military compaigne led by <persName ref="PRS7484Napier"></persName>.
```
Comment (thea-m) [inline PRS7001-8000/PRS7019MercaWa.xml]: campaign

**Comment by eu-genia** [comment]
Is PRS12796WaldaM different from PRS9946WaldaMi? Possible to have something to disambiguate?
Could you also add something on PRS12857Hagos to better disambiguate from other dagg. Hagos such as PRS4994hagos/PRS4997hagosTa etc.?

**Comment by eu-genia** [comment]
In PRS6668Mangasa:         "Added information and bibliographz"-->bibliography

File: PRS7001-8000/PRS7019MercaWa.xml [hunk 9]
```diff
@@ -44,26 +44,53 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+Mǝrč̣ā Warqe established a contact between <persName ref="PRS7484Napier"></persName> and <persName ref="PRS10378Yohanne"></persName>,
```
Comment (thea-m) [inline PRS7001-8000/PRS7019MercaWa.xml]: established contact, I think

File: new/PRS12857Hagos.xml [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+            <change who="DE" when="2020-01-02">Created entity</change>
+            <change when="2020-01-07" who="DE">Changed language of a personal name.</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="ti" xml:id="n1">ሐጎስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Ḥagos</persName>
```
Comment (thea-m) [inline new/PRS12857Hagos.xml]: also ti :)

**Comment by DariaElagina** [comment]
@eu-genia  I can't add any additional information on these persons, I just have titles and names

**Comment by DariaElagina** [comment]
@thea-m all done

**Pull request merged**